native/src/modules/endpoint/sendTrackingSignal.js

```diff
@@ -0,0 +1,46 @@
+// @flow
+
+import { createTrackingEndpoint, type SpecificSignalType } from 'api-client'
+import AppSettings from '../settings/AppSettings'
+import moment from 'moment'
+import type { SettingsType } from '../settings/AppSettings'
+import buildConfig from '../app/constants/buildConfig'
+
+const sendTrackingSignal = async ({ signal, offline = false }: {| signal: SpecificSignalType, offline?: boolean |}) => {
+  try {
+    const appSettings = new AppSettings()
+    const settings: SettingsType = await appSettings.loadSettings()
+    const {
+      jpalTrackingEnabled,
+      selectedCity,
+      contentLanguage,
+      allowPushNotifications,
+      proposeNearbyCities,
+      errorTracking,
+      jpalTrackingCode
+    } = settings
+
+    if (buildConfig().featureFlags.jpalTracking && jpalTrackingEnabled && jpalTrackingCode) {
+      await createTrackingEndpoint().request({
+        ...signal,
+        trackingCode: jpalTrackingCode,
+        metadata: {
+          offline,
+          timestamp: moment().toISOString(),
+          currentCity: selectedCity,
+          currentLanguage: contentLanguage,
+          systemLanguage: '', // TODO IGAPP-566 Include system language
+          appSettings: {
+            allowPushNotifications,
+            proposeNearbyCities,
+            errorTracking
+          }
+        }
+      })
+    }
+  } catch (e) {
+    console.error(e)
```

Comment:
Should we maybe send this to sentry?

Reply:
Also not sure whether console.error crashes the app in release mode.

Reply:
How can I do that? And no, it does not crash the app in release mode afaik but will test again. We have console.errors on quite a few places actually.